### PR TITLE
First try at pyqtads at conda-forge.

### PR DESCRIPTION
**Pull request by hhslepicka**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hhslepicka** [comment]
@conda-forge/help-python-c this one is good for a review.
Thanks!

File: recipes/pyqtads/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  license_file: LICENSE.md
+  summary: 'Advanced Docking System for Qt'
+
+  description: 'Advanced Docking System for Qt' 
+  dev_url: https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System
+
+extra:
+  recipe-maintainers:
+    - hhslepicka
+    - klauer
```
Comment (marcelotrevisani) [inline recipes/pyqtads/meta.yaml]: @klauer , do you agree to maintain this recipe as well?

Comment (klauer) [reply recipes/pyqtads/meta.yaml]: Yes, I will help with it.

**Comment by hhslepicka** [comment]
@marcelotrevisani any other comment?

File: recipes/pyqtads/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - pyqt >=5.9
+    - qt 5.12.*
```
Comment (scopatz) [inline recipes/pyqtads/meta.yaml]: are these really build deps?

Comment (hhslepicka) [reply recipes/pyqtads/meta.yaml]: Yes. This library compiles extensions that depends on Qt and Sip...

Comment (hhslepicka) [reply recipes/pyqtads/meta.yaml]: and pyqt is used also at the setup.py to query the Qt configuration and create resource files.

**Pull request merged**